USER_GUIDE.md

```diff
@@ -108,6 +109,90 @@ client
     .await?;
 ```
 
+### Make Raw Json Requests
+
+To invoke an API that is not supported by the client, use `client.send` method to do so. see [examples/json](./opensearch/examples/json.rs) for a complete working example
+
+#### PUT
+```rust
+// create index
+let response = client
+    .send(
+        Method::Put,
+        "/movies",
+        HeaderMap::new(),
+        Option::<&String>::None,
+        Option::<&String>::None,
+        None,
+    )
+    .await?;
+```
+
+#### POST
+
+```rust
+// Add a document
+let document =
+    r#"{
+      "name" : "Mission Impossible",
+      "year" : "2001"
+    }"#;
+
+let response = client
+    .send(
+        Method::Post,
+        "/movies/_doc",
+        HeaderMap::new(),
+        Option::<&String>::None,
+        Some(document),
+        None,
+    )
+    .await?;
+
+assert_eq!(response.status_code().as_u16(), 201_u16);
+```
+
+#### GET
+```rust
+// query document
+let query = json!({
+    "query": {
+      "match": {
+           "name" : "Mission Impossible"
+       }
+  }
+});
+
+let response = client
+    .send(
+        Method::Get,
+        "/movies/_search",
+        HeaderMap::new(),
+        Option::<&String>::None,
+        Some(query.to_string()),
+        None,
+    )
+    .await?;
+  
+assert_eq!(response.status_code().as_u16(), 200_u16);
```

Comment:
Does this work? Looks like a `GET` request with a query string, whereas OpenSearch uses `POST /.../_search` usually. 

Look at the response and print some search results?

Reply:
According to [the docs](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/searching.html), both work. The http client we use supports GET requests with a body, either way, i've changed to a POST instead